odw/services/data-warehouse/docs/walkthrough.md

```diff
@@ -0,0 +1,117 @@
+
+- **Data Models**: Pydantic `BaseModel` classes with `Key[str]` primary keys and proper type annotations
+- **IngestPipeline**: Six pipelines (source + final) with ingest, stream, table, and DLQ configurations  
+- **Stream Transformations**: Real-time data processing with `.add_transform()` and timestamp addition
+- **ConsumptionApi**: Type-safe HTTP endpoints using ClickHouse native parameter binding
+- **Workflow**: Temporal-powered workflows for pull-based data extraction with failure simulation
+- **Dead Letter Queue**: Automatic error handling with DLQ recovery transforms
+
+## Landing Page
+
+When the Data Warehouse frontend launches, users see the Connector Analytics page displaying three data sources: Blob, Log, and Event. Initially empty, clicking "Refresh" triggers Moose workflows that extract data via connectors, process it through real-time streams with transformations, and store it in ClickHouse tables. The page then displays unified metrics and recent data from all three sources.
+
+![landing page](./dw-images/dw-landing.png)
+
+![landing page populated](./dw-images/dw-landing-2.png)
+
+The Connector Analytics Report provides a unified dashboard view of all data sources. Three metric cards display real-time counts for Blob, Log, and Event connectors. The "Recent Data Summary" table shows the latest 10 records across all sources with key details like file names, sizes, and permissions for blobs. Users can trigger fresh data extraction by clicking the "Refresh" button, which populates the dashboard with current metrics and recent data from the warehouse.
+
+## Navigation
+
+Under the "Connectors" section we have links for All, Blobs, Logs and Events. All shows overview data for all connections and individual connector pages show specific information for each connector.
+
+![navbar](./dw-images/navbar.png)
+
+## Connector Pages
+
+### What Are Connectors?
+
+Connectors are mock data generators that simulate real-world data sources (blob storage, application logs, user events) for the data warehouse. They use a factory pattern to create domain-specific extractors that generate structured test data. In the data warehouse, Moose workflows use these connectors to extract data, which then flows through ingest pipelines for real-time processing and storage. Each connector produces Pydantic models that match the source data schemas, enabling type-safe data flow from extraction through transformation to final storage.
+
+### All Page
+
+The `All` connectors page displays a unified dashboard showing combined data from all three connectors. It features metric cards displaying real-time counts for Blob, Log, and Event data sources, plus a "Recent Data Summary" table showing the latest 10 records across all sources. Users can trigger fresh data extraction for all connectors simultaneously using the "Refresh" button, which populates the dashboard with current metrics and recent data from the warehouse.
```

Comment:
This description doesn't quite match the screenshot. This sounds like the description for connector analytics page.